specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/AvailableWorkloadProfiles.json

```diff
@@ -0,0 +1,133 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2022-06-01-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.App/locations/{location}/availableManagedEnvironmentsWorkloadProfileTypes": {
+      "get": {
```

Comment:
>get [](http://example.com/codeflow?start=7&length=3)

You can prepare for pagination by adding x-ms-pageable extension. That allows you to support it later without a breaking change or bumping the API version.